src/plugin/context/systemparams/rapier_context_systemparam.rs

```diff
@@ -35,21 +35,21 @@ pub struct ReadRapierContext<'w, 's, T: query::QueryFilter + 'static = With<Defa
 }
 
 impl<'w, 's, T: query::QueryFilter + 'static> ReadRapierContext<'w, 's, T> {
-    /// Use this method if you only have one [`RapierContext`].
+    /// Returns a single [`RapierContext`] corresponding to the filter (T) of [`ReadRapierContext`].
     ///
-    /// SAFETY: This method will panic if its underlying query fails.
+    /// If the number of query items is not exactly one, a [`bevy::ecs::query::QuerySingleError`] is returned instead.
     ///
-    /// Use the underlying query [`ReadRapierContext::rapier_context`] for safer alternatives.
-    pub fn single(&self) -> RapierContext {
+    /// You can also use the underlying query [`WriteRapierContext::rapier_context`] for finer grained queries.
```

Comment:
```suggestion
    /// You can also use the underlying query [`ReadRapierContext::rapier_context`] for finer grained queries.
```
I think I made a mistake here.